stl/inc/chrono

```diff
@@ -5451,6 +5497,28 @@ namespace chrono {
         }
         return _Os << static_cast<unsigned int>(_Val) << _STATICALLY_WIDEN(_CharT, " is not a valid month");
     }
+
+    template <class _CharT, class _Traits>
+    basic_ostream<_CharT, _Traits>& operator<<(basic_ostream<_CharT, _Traits>& _Os, const year& _Val) {
+        const tm _Time = {.tm_year = static_cast<int>(_Val)};

```

Comment:
```suggestion
        const tm _Time = {.tm_year = static_cast<int>(_Val) - 1900};
```

Reply:
Related, all the tests using `year` appear to be off by 1900.

Reply:
For some reason I thought `year` modelled from 1900, but I guess that's just `tm`.

Reply:
It's confusing, no doubt. I went through the same thing with [time.parse] while unpacking `tm` data after calling `get_time`.